ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -233,7 +233,9 @@ library
                        Ouroboros.Consensus.Storage.LedgerDB.HD.BackingStore
                        Ouroboros.Consensus.Storage.LedgerDB.HD.BackingStore.InMemory
                        Ouroboros.Consensus.Storage.LedgerDB.HD.BackingStore.Trivial
+                       Ouroboros.Consensus.Storage.LedgerDB.HD.DbChangelog
```

Comment:
One thing I realized: we're using the term `HD` to refer to hard-drive. Although SSDs are technically HDs, do we need this `HD` at all?

Reply:
No, not really. But it helps us divide the in memory part and the on disk part. Although I'm fine with renaming it. Suggestions?

Reply:
What about `Disk` or `Persistence`.  But maybe we want to do it later on. Leave it as is for now.

Reply:
Maybe these modules don't need to live in a submodule like `HD`: most of the `LedgerDB` code is going to be adapted to UTxO-HD anyway, so the separation between what is "HD" and what is not "HD" is not so straightforward. We could remove the indirection, and keep the `BackingStore` separation